package.json

```diff
@@ -30,7 +30,7 @@
         "validate:samples": "npm run build && npm --prefix samples install && npm run build:samples",
         "build:nolint": "npm run clean && npm run stripInternalDocs && echo Done",
         "stripInternalDocs": "tsc --pretty -p tsconfig.nocomments",
-        "test": "npm run validate:samples && npm run build && mocha --recursive ./lib/test/**/*-spec.js",
+        "test": "npm run build && mocha --recursive ./lib/test/**/*-spec.js",

```

Comment:
This change is just because one of our unit-testing sample doesn't play well with re-building it over and over. We need to do something about that, bu tI think that's outside the scope of this PR